typings/index.d.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+interface Props {
+  allowDuplicates?: boolean;
+  blurBehavior?: 'clear' | 'add' | 'ignore';
+  chipRenderer?: (...args: any[]) => any;
+  clearInputValueOnChange?: boolean;
+  dataSource?: any[];
+  dataSourceConfig?: {
+    text: string;
+    value: string;
+  };
+  defaultValue?: any[];
+  disabled?: boolean;
+  FormHelperTextProps?: object;
+  fullWidth?: boolean;
+  fullWidthInput?: boolean;
+  helperText?: React.ReactNode;
+  InputLabelProps?: object;
+  InputProps?: object;
+  inputRef?: (...args: any[]) => any;
+  label?: React.ReactNode;
+  newChipKeyCodes?: number[];
+  onAdd?: (...args: any[]) => any;
```

Comment:
These args are actually not "any" args. They are documented in the readme, we could make them more detailed.